### PR TITLE
Fixed initializer returns doc for implicitly unwrapped optional inits.

### PR DESCRIPTION
**Pull request by mohpor**

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/ValidDocsRule.swift [hunk 1]
```diff
@@ -66,7 +66,7 @@ func matchOutsideBraces(declaration: String) -> NSString? {
 }
 
 func declarationIsInitializer(declaration: String) -> Bool {
-    return !regex("^((.+)?\\s+)?init\\?*\\(.*\\)")
+    return !regex("^((.+)?\\s+)?init\\?*\\!*\\(.*\\)")
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/ValidDocsRule.swift]: I think `\\?*\\!*` should be `[\\?\\!]{0,1}`.

**Comment by jpsim** [comment]
Could you please add a bug fix entry to the changelog? Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -97,7 +99,7 @@ This release has seen a phenomenal uptake in community contributions!
   [bootstraponline](https://github.com/bootstraponline)
   [#689](https://github.com/realm/SwiftLint/issues/689)
 
-* Add rule to check for private unit tests (private unit tests don't get run 
+* Add rule to check for private unit tests (private unit tests don't get run
```
Comment (keith) [inline CHANGELOG.md]: This trailing white space is required for the new lines.

File: Source/SwiftLintFramework/Rules/ValidDocsRule.swift [hunk 3]
```diff
@@ -66,7 +66,7 @@ func matchOutsideBraces(declaration: String) -> NSString? {
 }
 
 func declarationIsInitializer(declaration: String) -> Bool {
-    return !regex("^((.+)?\\s+)?init\\?*\\(.*\\)")
+    return !regex("^((.+)?\\s+)?init[\\?\\!]{0,1}\\(.*\\)")
```
Comment (keith) [inline Source/SwiftLintFramework/Rules/ValidDocsRule.swift]: Can you add a test case below for this new behavior?

File: CHANGELOG.md (hunk 2 above)
Comment (mohpor) [inline CHANGELOG.md]: I didn't check other PR's trailing white spaces. I'll double check.

File: Source/SwiftLintFramework/Rules/ValidDocsRule.swift (hunk 3 above)
Comment (mohpor) [inline Source/SwiftLintFramework/Rules/ValidDocsRule.swift]: I can try.

**Comment by marcelofabri** [comment]
Closing this PR since we removed this rule. Thanks for contributing!

**Pull request closed without merging**